Main/StackWalker/StackWalker.cpp

```diff
@@ -87,162 +87,36 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <tchar.h>
-#include <windows.h>
 #pragma comment(lib, "version.lib") // for "VerQueryValue"
 #pragma warning(disable : 4826)
 
+#ifdef UNICODE
+  #define DBGHELP_TRANSLATE_TCHAR
 
-// If VC7 and later, then use the shipped 'dbghelp.h'-file
+#endif
 #pragma pack(push, 8)
-#if _MSC_VER >= 1300

```

Comment:
This will remove the support of older compilers... there is no need to remove this...